charts/ccd-data-store-api/values.yaml

```diff
@@ -25,7 +25,7 @@ java:
     DATA_STORE_DB_USERNAME: ccd@ccd-data-store-api-postgres-db-{{ .Values.global.environment }}
     DATA_STORE_DB_OPTIONS: "?stringtype=unspecified&sslmode=require"
     DATA_STORE_DB_MAX_POOL_SIZE: 48
-
+    LOG_CALLBACK_DETAILS:
```

Comment:
shall we add a quick comment that explain what this does?

Reply:
added explanation